crds/run_v1beta1_runjob.yaml

```diff
@@ -602,6 +602,49 @@ spec:
                         description: A list of Volumes to make available to containers.
                         items:
                           properties:
+                            cloudSqlInstance:
+                              description: For Cloud SQL volumes, contains the specific
+                                instances that should be mounted. Visit https://cloud.google.com/sql/docs/mysql/connect-run
+                                for more information on how to connect Cloud SQL and Cloud
+                                Run.
+                              properties:
+                                instances:
```

Comment:
IIUC this file is generated and field name should be updated to `instanceRefs` as well?

Reply:
Good catch! I missed it running `make manifests`, fixed with https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1688/commits/26dbc1818468f9b6ea10c7f6a248bf0b07ed1f0d